activestorage/lib/active_storage/engine.rb

```diff
@@ -93,7 +94,7 @@ class Engine < Rails::Engine # :nodoc:
     initializer "active_storage.services" do
       ActiveSupport.on_load(:active_storage_blob) do
         if config_choice = Rails.configuration.active_storage.service
-          configs = Rails.configuration.active_storage.service_configurations ||= begin
+          Rails.configuration.active_storage.service_configurations ||= begin
```

Comment:
Hm, I'm getting error:
```
NoMethodError (Cannot configure service :local for Post#avatar`:
undefined method `deep_symbolize_keys' for nil:NilClass):
```
We should ensure that this block is always executed before `ActiveStorage::ServiceRegistry.fetch(service_name)`  call

Reply:
Thanks for catching that! Moved config reading to the ServiceRegistry